daq_macros.py

```diff
@@ -2259,11 +2283,11 @@ def gotoMaxRaster(rasterResult,multiColThreshold=-1,**kwargs):
     logger.info("goto " + str(x) + " " + str(y) + " " + str(z))
 
     if 'omega' in kwargs:
-      beamline_lib.mvaDescriptor("sampleX",x,
-                                 "sampleY",y,
-                                 "sampleZ",z,
+      beamline_lib.mvaDescriptor("sampleX",x/1000,
```

Comment:
there are a lot of these unit changes - can you abstract them so that they don't make destructive changes to AMX/FMX?

Reply:
These changes are being made directly because they will not be kept in LSDC after the shutdown. The MD2 code for rastering will be completely different, and none of these changes are expected to remain.